src/Symfony/Component/Finder/Tests/GitignoreTest.php

```diff
@@ -442,4 +442,71 @@ public function providerExtended(): array
 
         return $cases;
     }
+
+    /**
+     * @dataProvider provideNegatedPatternsCases
+     */
+    public function testToRegexMatchingNegatedPatterns(array $gitignoreLines, array $matchingCases, array $nonMatchingCases)
```

Comment:
better to put just right below `testToRegex` method as almost the same

Reply:
I prefer not mixing tests and their data providers.